docs/reference/api/stravalib.field_conversions.enum_value.rst

```diff
@@ -0,0 +1,6 @@
+﻿stravalib.field\_conversions.enum\_value
```

Comment:
You can ignore all pages in this api/ dir. they are automagically generated when you build the docs.